File: Go/vowels.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	sentence := "The quick brown fox jumps over the lazy dog"

	var VOWELS = "AEIOUYaeiouy"

	// Split the string into an array
	sentenceArray := strings.Split(sentence, "")

	// Initiate the counter
	var count int = 0

	// Loop through the string
	for i := 0; i < len(sentenceArray); i++ {
		// Look for the index of the current character in the VOWELS string
		index := strings.Index(VOWELS, sentenceArray[i])

		if index != -1 {
			count++
		}
	}

	// Print the result
	fmt.Println(count)

}

File: PHP/vowels.php
<?php

$sentence = "The quick brown fox jumps over the lazy dog";
$VOWELS = "AEIOUYaeiouy";

// Split sentence to array
$sentence_array = str_split($sentence);

// Initialize counter
$counter = 0;

// Loop through sentence array
for($i = 0; $i < count($sentence_array); $i++){
    // Check if the current character exists in the vowels-string
    if(str_contains($VOWELS, $sentence_array[$i])){
        $counter = $counter + 1;
    }
}

// Print result
echo"{$counter}";

?>